arroyo-sql-macro/src/lib.rs

```diff
@@ -209,6 +209,7 @@ impl Parse for PipelineCase {
 
 struct CorrectnessTestCase {
     test_name: LitStr,
+    checkpoints: LitStr,
```

Comment:
You can make this a `syn::LitBool` and then your macros can just take `true` and `false`.

Reply:
Done